akirachix/Upile-Informational
# Feature/partners

## What this PR does?
This PR updates the "partners" section of the informational website to display our partners.

Perhaps answering the following questions will help create a good description:
* What is this?
This update enhances the partners section to showcase our partner organizations.

* Why this?
 It highlights our partners and provide a more engaging user experience.

* How are you doing this? 
Created a new section in the informational website to display the partners.

- List of updates for features, enhancements, or bugfixes
Added a new partners section to the application.

## Type of change
New feature

Specify the kind of PR/MR

Please delete options that are not relevant.


## How can it be tested?
Navigate to the Partners Section
Ensure that all partner logos are correctly displayed
Verify that the partners section is responsive and displays correctly on different screen sizes and devices.
![Screenshot from 2024-09-14 11-24-48](https://github.com/user-attachments/assets/3f1f7a04-50be-44fd-a55e-787c7aa5a210)



# Where is it documented?
https://wambuinatalygakenye.atlassian.net/browse/UP-135?atlOrigin=eyJpIjoiZmM4MzVjZTJiODU4NGU3NWJhODA4MDM4MWIxOGQ3NTUiLCJwIjoiaiJ9
